### PR TITLE
:sparkles: Add Sanity CMS

### PR DESCRIPTION
**Pull request by rmtuckerphx**
## Proposed Changes

Add CMS Integration for Sanity

## Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project
- [x] My change requires a change to the documentation
- [x] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [x] All new and existing tests passed

### CODE REVIEW

**Comment by jankoenig** [comment]
Thank you @rmtuckerphx! 🎉 

 We'll take a look this week

**Comment by jankoenig** [comment]
Thanks a lot for the great PR and documentation @rmtuckerphx 🎉 

I updated the docs a bit and also added `query: "*[_type == 'translation' && !(_id in path('drafts.**'))]",` as default `query` for `TranslationsQueryTransformer`. GROQ can look a bit intimidating at first, so I tried to reduce that a bit for a common use case like translations. Does this make sense?

Thanks!
Jan

File: integrations/cms-sanity/docs/README.md [hunk 1]
```diff
@@ -93,96 +115,143 @@ new SanityCms({
...
 
 ### apiVersion
 
-Specifies the version of the API. Read [API Versioning](https://www.sanity.io/docs/api-versioning) for more details. By default, uses the current date: `new Date().toISOString().split('T')[0]`
+Specifies the version of the API.
+
+```typescript
+new SanityCms({
+  client: {
+    projectId: '<YOUR-PROJECT-ID>',
```
Comment (rmtuckerphx) [inline integrations/cms-sanity/docs/README.md]: This should be an example of apiVersion

**Comment by rmtuckerphx** [comment]
@jankoenig Setting the default query for TranslationsQueryTransformer makes sense.

Added 1 comment about docs.
Otherwise, ready to go.

**Review by jankoenig — APPROVED** [review]
Thank you @rmtuckerphx!

**Review by aswetlow — APPROVED** [review]

**Pull request merged**